src/Raven.Studio/typescript/models/database/index/indexFieldOptions.ts

```diff
@@ -213,15 +221,24 @@ class indexFieldOptions {
         }
     }
     
+    setStorageAccordingToEngine(engine: string) {
```

Comment:
do we need this method? looks like we have engine available as observable in this class.